.github/workflows/windows.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     uses: ./.github/workflows/commit_message.yml
 
   test:
-    name: cp312 (meson) fast
+    name: fast, py3.12/npAny, dev.py
```

Comment:
I didn't find anything particularly notable about this job or the next. Maybe I'll use these for timing and `fail_slow`?

Reply:
Sure, one Windows job for that seems fine - and this one is pretty generic.